index.php

```diff
@@ -28,6 +28,35 @@
 
 </section>
 
+<div class="boxes">
+
+    <form>
+        Alignment:<br>
+        <input type="text" name="alignment">
+    </form>
+
+    <form>
```

Comment:
Put all the inputs inside a single form tag. If you need to manipulate the different parts of the form, contain them in separate div tags rather than form times, as otherwise you'll have a ton of posts to deal with!